applications/tari_validator_node/src/http_ui/server.rs

```diff
@@ -22,15 +22,24 @@
 
 use std::{net::SocketAddr, sync::Arc};
 
-use axum::{routing::get, Extension, Router};
+use axum::{routing::get_service, Extension, Router};
 use log::{error, info};
+use reqwest::StatusCode;
+use tower_http::services::ServeDir;
 
 const LOG_TARGET: &str = "tari_validator_node::http_ui::server";
 
 pub async fn run_http_ui_server(address: SocketAddr, json_rpc_address: Option<String>) -> Result<(), anyhow::Error> {
-    let router = Router::new().route("/", get(index)).layer(Extension(Arc::new(
-        json_rpc_address.unwrap_or_else(|| "127.0.0.1:18145".to_string()),
-    )));
+    let serve_dir_service = get_service(ServeDir::new(
+        "d:/code/work/tari-dan/applications/tari_validator_node_web_ui/build",
```

Comment:
Can we use a relative path here?
We could also include the built assets directly in the binary as per
https://github.com/tari-project/tari-dan/issues/86

Reply:
Oh, I forgot about that. I will look into the `include_dir`. I also need to pass somehow the json_rpc port from the VN to the react app.

Reply:
Yeah that's tricky, maybe create a config struct, serve that struct as json at `/config.json`. Or you could string replace e.g. `// $CONFIG$` with `const CONFIG = {....}` in the compiled js

Reply:
I added a special route `/json_rpc_address` to the http server that provides the port back to the react ui. This could be avoided if we decide to merge jrpc and http to the same server, I don't see a reason why not. The jrpc can be served from a special route e.g. all request to /jrpc could be served by that handlers. But I'm not sure if we have a reason to run two servers.